app/graphql/types/outputs/ont/flowcell_type.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Types
+  module Outputs
+    module Ont
+      # The type for Ont::Flowcell objects.
+      class FlowcellType < CommonOutputObject
+        field :position, Integer, null: false
+        field :uuid, String, null: false
```

Comment:
I don't think the UI requires the uuid, but cant harm in keeping it in the api?

Reply:
Sure .. I was just putting the fields from the underlying Rails model so you can get them if you want and, if you don't, you don't ask for them.